tests/utils.py

```diff
@@ -26,14 +27,17 @@ def run(
         benchmarks,
     ):
 
-    with benchmarks:
+    history = HistoryCallback()
+
+    with benchmarks(history):
```

Comment:
Does it make sense to have the instantiated history callback object be the one passed around here? This is replacing the `client` in coiled benchmarks.